csi/volumeutils.py

```diff
@@ -2,20 +2,20 @@
 Utility functions for Volume management
 """
 
-import os
 import json
-import time
 import logging
-import threading
+import os
 import shutil
+import threading
+import time
 from errno import ENOTCONN
 
 from jinja2 import Template
 
-from kadalulib import execute, PV_TYPE_SUBVOL, PV_TYPE_VIRTBLOCK, \
-    get_volname_hash, get_volume_path, logf, makedirs, CommandException, \
-    retry_errors, is_gluster_mount_proc_running, SizeAccounting
-
+from kadalulib import (PV_TYPE_SUBVOL, PV_TYPE_VIRTBLOCK, CommandException,
+                       SizeAccounting, execute, get_volname_hash,
+                       get_volume_path, is_gluster_mount_proc_running, logf,
+                       makedirs, retry_errors)
```

Comment:
doesn't pylint capture these formatting issues?

Reply:
Then, should we mandate everyone using `isort` before submit? By running make isort test?

Reply:
@amarts missed your suggestion earlier,
- `pylint`/`flake8` doesn't modify code and a good candidate for `make` (CI in general).
- However `isort` and `yapf` modifies the code and a good candidate to be stated in contrib guide.
- If we want to include linters & fixers in contrib guide and make full use of them, I feel either you or @aravindavk run `yapf` and `isort` on whole code base.
- I can also do this with your agreement however it results in a huge diff on my name 😅